L1Trigger/L1THGCal/plugins/geometries/HGCalTriggerGeometryV9Imp3.cc

```diff
@@ -68,6 +68,12 @@ class HGCalTriggerGeometryV9Imp3 : public HGCalTriggerGeometryBase {
   unsigned hSc_triggercell_size_ = 2;
   unsigned hSc_module_size_ = 12;  // in TC units (144 TC / panel = 36 e-links)
   unsigned hSc_wafers_per_module_ = 3;
+  unsigned hSc_modules_per_sector_ = 4;
+  unsigned hSc_front_layers_split_ = 12;
+  unsigned hSc_back_layers_split_ = 8;
+  unsigned hSc_layer_for_split_ = 40;
+  int hSc_tc_layer0_min_ = 24;
+  const int ntc_per_wafer = 48;
```

Comment:
It seems that these variables are not modified. If that's the case they should be defined as `static constexpr`

Reply:
Done, apart from the first 2 which can be set in the config file